### ISSUE TITLE
Choosing between Tailwind and Material UI for styling in a project

### ISSUE THREAD

**Issue by ank1traj** [ank1traj/testcase-generator#44]
We need to decide which styling library to use for our project, and we are considering either Tailwind or Material UI. Both of these libraries have their own advantages and disadvantages, and we need to weigh them carefully before making a decision.

Tailwind is a utility-first CSS framework that makes it easy to create custom designs without writing CSS from scratch. It offers a wide range of pre-designed classes that can be combined to create a unique design, and it has a minimal footprint.

Material UI, on the other hand, is a component library that provides a set of pre-designed components that can be used to create a consistent look and feel throughout the application. It offers a range of customizable components, and it has a strong focus on accessibility.

**Comment by ank1traj** [comment]
@tayyab-ilyas Yes, I am looking for tailwind. But some people are more comfortable with Material UI. So, both are open. Whoever wanna work on this issue can pick any one.

**Comment by mud1tx** [comment]
@ank1traj I can work on this issue using TailWind and will make the whole website responsive.Please assign it to me.

**Comment by mud1tx** [comment]
@ank1traj Completed 10%, was working on some college stuff just need another 3-4 days to complete from today. Sorry for the delay.

**Comment by ank1traj** [comment]
Hi @mud1tx

Any update on this?

**Comment by PSS2134** [comment]
@ank1traj I can work on this

**Comment by ank1traj** [comment]
Please
